_licenses/mulanpsl-2.0.txt

```diff
@@ -0,0 +1,225 @@
+---
+title: Mulan Permissive Software License，Version 2
+spdx-id: MulanPSL-2.0
+hidden: true
+
+description: A permissive license similar to the <a href="/licenses/apache-2.0/">Apache License</a>, but explicitly states that it does not grant trademark rights. Mulan Permissive Software License is the first open source license in both Chinese and English approved by OSI.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+note: It's suggested to take the additional step of adding a boilerplate notice to the top of each file. The boilerplate can be found at the end of the license.
+
+using:
+  - openGauss Server: https://github.com/opengauss-mirror/openGauss-server/blob/master/LICENSE
+  - OSCPU NutShell: https://github.com/OSCPU/NutShell/blob/master/LICENSE
+  - Android CN OAID: https://github.com/gzu-liyujiang/Android_CN_OAID/blob/master/LICENSE
```

Comment:
Drop the `- ` before each example, see https://github.com/github/choosealicense.com/pull/779

This is what is causing CI to fail.